examples/mlops-env-on-gcp/creating-notebook-instance/exercises/README.md

```diff
@@ -0,0 +1,107 @@
+# Creating an AI Platform Notebooks instance
+
+**Learning Objectives:**
+ 1. Write a Dockerfile to define a custom image
+ 2. Build and push the image to your Container Registry
+ 2. Create an AI Platform Notebook using a custom container image
+ 
+In this lab, you will provision an AI Platfom Notebooks instance using a custom container image.
+
+The accompanying lab - `provisioning-kfp` - describe the steps to provision other services in the MLOps environment, including a standalone deployment of Kubeflow Pipelines.
+
+## Enabling the required cloud services
+
+In addition to the [services enabled by default](https://cloud.google.com/service-usage/docs/enabled-service), the following additional services must be enabled to provision an instance of **AI Platform Notebooks**:
+
+1. Compute Engine
+1. Container Registry
+1. Cloud Build
+
+Use [GCP Console](https://console.cloud.google.com/) or `gcloud` command line interface in [Cloud Shell](https://cloud.google.com/shell/docs/) to [enable the required services](https://cloud.google.com/service-usage/docs/enable-disable) . 
+
+You can enable the required services using `gcloud`:
+1. Start GCP [Cloud Shell](https://cloud.google.com/shell/docs/)
+2. Make sure that **Cloud Shell** is configured to use your project. In Cloud Shell, bype the following commands. Replace `[YOUR_PROJECT_ID]` with your GCP Project ID.
+
+```
+PROJECT_ID=[YOUR_PROJECT_ID]
+
+gcloud config set project $PROJECT_ID
+```
+
+3. Enable services
+```
+gcloud services enable \
+compute.googleapis.com \
+container.googleapis.com \
+cloudbuild.googleapis.com 
+
+```
+
+## Creating an **AI Platform Notebooks** instance
+
+You will use a custom container image with KFP and TFX SDKs pre-installed to create your instance. 
+
+### Building a custom docker image:
+
+1. In **Cloud Shell**,  create a working folder in your `home` directory
+```
+cd
+mkdir lab-workspace
+cd lab-workspace
+```
+
+2. Create the requirements file with the Python packages to deploy to your instance
+```
+cat > requirements.txt << EOF
+pandas<1.0.0
+click==7.0
+tfx==0.21.2
+kfp==0.2.5
```

Comment:
I would recommend updating the package versions to the following:
tfx==0.21.4
kfp==0.5.1